File: branch_cyberSec/piscine/day_8_vaccine/src/logger.js
import { styleText } from 'node:util';

export const print_info = (...a) => console.log(styleText('yellow', "[INFO]"), ...a);
export const print_error = (...a) => console.error(styleText('red', "[ERROR]"), ...a);
export const print_success = (...a) => console.log(styleText('green', "[SUCCESS]"), ...a);
export const print_fatal = (...a) => console.error(styleText(['magenta', 'bold'], "[FATAL]"), ...a);

File: CyberSec/piscine/day_2_ft_otp/logger.py
from colorama import Fore

def error(message: str) -> None:
    print(f"[{Fore.RED}ERROR{Fore.RESET}]", message)

def info(message: str) -> None:
    print(f"[{Fore.YELLOW}INFO{Fore.RESET}]", message)

def success(message: str) -> None:
    print(f"[{Fore.GREEN}SUCCESS{Fore.RESET}]", message)